bot/exts/evergreen/latex.py

```diff
@@ -64,12 +63,13 @@ async def latex(self, ctx: commands.Context, *, text: str) -> None:
         """Renders the text in latex and sends the image."""
         text = self._prepare_input(text)
         async with ctx.typing():
-            image = self._render(text)
 
-            if isinstance(image, BytesIO):
-                await ctx.send(file=discord.File(image, "latex.png"))
-            else:
-                await ctx.send("```" + image + "```")
+            with ThreadPoolExecutor() as pool:
+                image = await asyncio.get_running_loop().run_in_executor(
+                    pool, partial(self._render, text)
```

Comment:
This doesn't need to be a partial since text is a normal arg, you'd only need a partial if using kwargs.